<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLinksDialog</class>
 <widget class="QDialog" name="AddLinksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Link</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../retroshare-gui/src/gui/images.qrc">
    <normaloff>:/images/rstray3.png</normaloff>:/images/rstray3.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="4">
    <widget class="HeaderFrame" name="headerFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="1" column="1">
       <widget class="QPushButton" name="closepushButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="addLinkButton">
        <property name="text">
         <string>Add Link</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>375</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Add a new Link</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Title:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Url:</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLineEdit" name="titleLineEdit"/>
             </item>
             <item>
              <widget class="QLineEdit" name="linkLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="buttonFrame">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="anonBox">
              <property name="text">
               <string>Add Anonymous Link</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scoreBox">
              <item>
               <property name="text">
                <string>+2 Great!</string>
               </property>
               <property name="icon">
                <iconset resource="../../retroshare-gui/src/gui/images.qrc">
                 <normaloff>:/images/filerating5.png</normaloff>:/images/filerating5.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>+1 Good</string>
               </property>
               <property name="icon">
                <iconset resource="../../retroshare-gui/src/gui/images.qrc">
                 <normaloff>:/images/filerating4.png</normaloff>:/images/filerating4.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>0 Okay</string>
               </property>
               <property name="icon">
                <iconset resource="../../retroshare-gui/src/gui/images.qrc">
                 <normaloff>:/images/filerating3.png</normaloff>:/images/filerating3.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>-1 Sux</string>
               </property>
               <property name="icon">
                <iconset resource="../../retroshare-gui/src/gui/images.qrc">
                 <normaloff>:/images/filerating2.png</normaloff>:/images/filerating2.png</iconset>
               </property>
              </item>
              <item>
               <property name="text">
                <string>-2 Bad Link</string>
               </property>
               <property name="icon">
                <iconset resource="../../retroshare-gui/src/gui/images.qrc">
                 <normaloff>:/images/filerating1.png</normaloff>:/images/filerating1.png</iconset>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>299</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QTextEdit" name="linkTextEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../retroshare-gui/src/gui/images.qrc"/>
 </resources>
 <connections/>
</ui>
